sdccc/src/main/java/com/draeger/medical/sdccc/messages/mapping/MessageContent.java

```diff
@@ -75,6 +75,9 @@ public class MessageContent {
     private String uuid;
     private boolean isSOAP;
 
+    @Column(name = "sender", length = 15)
```

Comment:
The length restriction will not work for future IPv6 support.

Please let it default to Derby's VARCHAR(255).

Reply:
removed the length restriction. Should now be able to store IPv6 addresses as well.